api/tests/integration/edge_api/identities/test_edge_identity_featurestates_view.py

```diff
@@ -134,6 +134,43 @@ def test_edge_identities_featurestate_delete(
     assert response.status_code == status.HTTP_204_NO_CONTENT
 
 
+def test_edge_identities_with_identifier_featurestate_delete(
+    admin_client,
+    environment,
+    environment_api_key,
+    identity_document,
+    dynamo_wrapper_mock,
+):
+    # Given
+    dynamo_wrapper_mock.get_item.return_value = identity_document
+    identifier = identity_document["identifier"]
+    featurestate_uuid = identity_document["identity_features"][0]["featurestate_uuid"]
```

Comment:
how do users get hold of the featurestate_uuid? Is this exposed somewhere?

Reply:
Yeah, that's sort of what the problem is - they have to use the old endpoint that uses the identity uuid, but then this sort of defeats the whole  point

Reply:
Yeah 🤔 is there a reason we can't add the option to use the name? They're unique per project anyway, right?

Reply:
Yeah, but that will break the consistency since it's ID that used everywhere? But Yeah, I guess we don't really have a choice